portindex2json/portindex2json.tcl

```diff
@@ -24,8 +96,13 @@ while {[gets $fd line] >= 0} {
     array set portinfo $line
     array unset json_portinfo
     foreach key [array names portinfo] {
-        if { $key == "categories"} {
-            set json_portinfo($key) [::json::write array {*}$portinfo($key)]
+        if { $key == "categories"  || $key=="variants" || $key=="depends_lib" || $key=="depends_build" || $key=="depends_fetch" } {
+            set json_portinfo($key) [::json::write array {*}[format_strings $portinfo($key)]]
```

Comment:
This is a bit of personal opinion, but I would rewrite this in such a way that `::json::write array` would already be called inside the `format_strings` function. Then you would only call something like
```tcl
set json_portinfo($key) [strings2json_array $portinfo($key)]
```
(I changed the function name from `format_strings` to  `strings2json_array`. Not sure if that's the best name either.)